httpcore/_async/connection_pool.py

```diff
@@ -96,6 +96,15 @@ def __init__(
         self._backend = AutoBackend()
         self._next_keepalive_check = 0.0
 
+        if http2:
+            try:
+                import h2
+            except ImportError:
+                raise ImportError(
+                    "Attempted to use http2=True, but the 'h2' "
+                    "package is not installed. Use 'pip install httpcore[http2]'."
+                )
```

Comment:
I assume we'll do the same dance in HTTPX when a client is given `http2=True`, such that an appropriate `ImportError` hinting at `httpx[http2]` is raised before this code is even reached?

Reply:
Yes I think so.